concepticondata/conceptlists/Bowern-2021-203c.tsv

```diff
@@ -0,0 +1,204 @@
+Bowern-2021-203c-56	56	deity/powerful spirit/culture figure 	720	culture-mythology	Culture Vocabulary	3231	DEITY
+Bowern-2021-203c-57	57	digging stick 	627	subsistence tool	Culture Vocabulary	339	DIGGING STICK
+Bowern-2021-203c-58	58	dipper 	628	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-59	59	dipper (small coolamon) 	777	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-60	60	dream 	629	culture-mythology	Culture Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-203c-61	61	dream	397	mental	Culture Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-203c-62	62	dreaming 	778	culture-mythology	Culture Vocabulary	1920	DREAM (SOMETHING)
+Bowern-2021-203c-63	63	drone pipe 	630	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-64	64	drum 	631	culture-mythology	Culture Vocabulary	908	DRUM
+Bowern-2021-203c-65	65	drum (large signal) 	829	culture	Culture Vocabulary		
+Bowern-2021-203c-66	66	earring 	633	other	Culture Vocabulary	770	EARRING
+Bowern-2021-203c-67	67	earth oven 	818	food	Culture Vocabulary		
+Bowern-2021-203c-68	68	fan 	721	subsistence tool	Culture Vocabulary	1971	FAN (OBJECT)
+Bowern-2021-203c-69	69	feather headband 	604	dress	Culture Vocabulary		
+Bowern-2021-203c-70	70	feather headdress 	722	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-71	71	fence, palisade 	742	other	Culture Vocabulary	1690	FENCE
+Bowern-2021-203c-72	72	fermented drink 	830	culture	Culture Vocabulary		
+Bowern-2021-203c-73	73	fetish, charm 	723	culture-mythology	Culture Vocabulary	2804	AMULET
+Bowern-2021-203c-74	74	fire drill 	635	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-75	75	fire-saw 	779	other	Culture Vocabulary		
+Bowern-2021-203c-76	76	fire-tender 	636	culture-mythology	Culture Vocabulary		
+Bowern-2021-203c-77	77	fireplace	637	other	Culture Vocabulary	185	FIREPLACE
+Bowern-2021-203c-78	78	firewood 	638	other	Culture Vocabulary	10	FIREWOOD
+Bowern-2021-203c-79	79	fish (with fish-poison) 	724	food	Culture Vocabulary		
+Bowern-2021-203c-80	80	fish (with line) 	725	food	Culture Vocabulary	2796	FISHING (WITH A HOOK)
+Bowern-2021-203c-81	81	fish (with spear) 	781	food	Culture Vocabulary		
+Bowern-2021-203c-82	82	fish hook 	643	subsistence tool	Culture Vocabulary	1074	FISHHOOK
+Bowern-2021-203c-83	83	fish poison 	641	subsistence tool	Culture Vocabulary	348	FISH POISON
+Bowern-2021-203c-84	84	fishing line 	642	subsistence tool	Culture Vocabulary	19	FISHING LINE
+Bowern-2021-203c-85	85	fishtrap 	726	subsistence tool	Culture Vocabulary	389	FISH TRAP
+Bowern-2021-203c-86	86	fishtrap (permanent) 	782	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-87	87	fishtrap (woven) 	783	subsistence tool	Culture Vocabulary		
+Bowern-2021-203c-88	88	flat bread, cassava bread 	727	food	Culture Vocabulary		
+Bowern-2021-203c-89	89	flour (bush) 	785	food	Culture Vocabulary		
+Bowern-2021-203c-90	90	flour (European) 	814	food	Culture Vocabulary		
+Bowern-2021-203c-91	91	flour/meal from manioc 	728	food	Culture Vocabulary		
```

Comment:
324	MANIOC FLOUR ?

Reply:
I'd leave it unmapped because of the "meal" aspect in there. Sounds more like generally anything that can be made from manioc.